homeassistant/components/generic/camera.py

```diff
@@ -93,6 +95,7 @@ def __init__(self, hass, device_info):
             self.stream_options[FFMPEG_OPTION_MAP[CONF_RTSP_TRANSPORT]] = device_info[
                 CONF_RTSP_TRANSPORT
             ]
+        self.stream_options[CONF_LL_HLS] = device_info.get(CONF_LL_HLS)
```

Comment:
The changes in generic just allow it to take in CONF_LL_HLS from the config and pass it along to the stream component through stream_options.